Common/Product/SharedProject/ProcessOutput.cs

```diff
@@ -237,27 +234,18 @@ sealed class ProcessOutput : IDisposable {
         ) {
             var outFile = Path.GetTempFileName();
             var errFile = Path.GetTempFileName();
-            var psi = new ProcessStartInfo("cmd.exe");
-            psi.CreateNoWindow = true;
-            psi.WindowStyle = ProcessWindowStyle.Hidden;
-            psi.UseShellExecute = true;
-            psi.Verb = "runas";
-
-            string args;
-            if (quoteArgs) {
-                args = string.Join(" ", arguments.Where(a => a != null).Select(QuoteSingleArgument));
-            } else {
-                args = string.Join(" ", arguments.Where(a => a != null));
-            }
-            psi.Arguments = string.Format("/S /C \"{0} {1} >>{2} 2>>{3}\"",
-                QuoteSingleArgument(filename),
-                args,
-                QuoteSingleArgument(outFile),
-                QuoteSingleArgument(errFile)
-            );
-            psi.WorkingDirectory = workingDirectory;
```

Comment:
I think my question is more: what testing would make you more confident in the change?

I generally understand what `UseShellExecute` and `WorkingDirectory` do, but also believe that, with this change, working directory is only going to be used directly for looking up `cmd.exe`. Once we actually run `node ...`, `pushd` has already changed the working directory.

I verified all the basic scenarios against this change, while also running node programs to look at `process.cwd()` and opening relative file paths. Any other thoughts?